webui/src/main/java/org/polypheny/db/webui/HttpServer.java

```diff
@@ -264,7 +264,7 @@ private void attachRoutes( Javalin webuiServer, Crud crud ) {
 
         webuiServer.get( "/getAvailableSources", crud::getAvailableSources );
 
-        webuiServer.post( "/createAdapter", crud::addAdapter );
+        webuiServer.post( "/createStore", crud::addAdapter );
```

Comment:
If there is now a `createStore`shouldn't there also be a `createSource`?

Reply:
Auto-rename also renamed crud method. Only catalog `DdlManager `method was split. Reverted.